clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java

```diff
@@ -318,11 +319,66 @@ public FileStatus getFileStatus(Path path) throws IOException {
 
     FileStatus ret = new FileStatus(fileStatus.getLength(), fileStatus.isIsFolder(),
         BLOCK_REPLICATION_CONSTANT, fileStatus.getBlockSizeBytes(), fileStatus.getCreationTimeMs(),
-        fileStatus.getCreationTimeMs(), null, null, null, new Path(mTachyonHeader + tPath));
+            fileStatus.getCreationTimeMs(), new FsPermission((short) fileStatus.getPermission()),
+            fileStatus.getUserName(), fileStatus.getGroupName(), new Path(mTachyonHeader + tPath));
     return ret;
   }
 
   /**
+   * Changes owner or group of a path (i.e. a file or a directory). If username is null, the
+   * original username remains unchanged. Same as groupname.
+   *
+   * @param path path to set owner or group
+   * @param username username to be set
+   * @param groupname groupname to be set
+   * @throws IOException if the path failed to be changes owner or group
+   */
+  @Override
+  public void setOwner(Path path, final String username, final String groupname)
+      throws IOException {
+    TachyonURI tPath = new TachyonURI(Utils.getPathWithoutScheme(path));
+    Path hdfsPath = Utils.getHDFSPath(tPath, mUnderFSAddress);
+    LOG.info("setOwner({},{},{}) HDFS Path: {} TPath: {}{}", path, username, groupname, hdfsPath,
+        mTachyonHeader, tPath);
+    try {
+      if (username != null && !username.isEmpty()) {
+        SetAclOptions options =
+            new SetAclOptions.Builder().setOwner(username).setRecursive(false).build();
+        mTFS.setAcl(tPath, options);
+      }
+      if (groupname != null && !groupname.isEmpty()) {
+        SetAclOptions options =
+            new SetAclOptions.Builder().setGroup(groupname).setRecursive(false).build();
+        mTFS.setAcl(tPath, options);
+      }
+    } catch (TachyonException e) {
+      throw new IOException(e);
+    }
+  }
+
+  /**
+   * Changes permission of a path.
+   *
+   * @param path path to set permission
+   * @param permission permission set to path
+   * @throws IOException if the path failed to be changed permission
+   */
+  public void setPermission(Path path, FsPermission permission) throws IOException {
+    TachyonURI tPath = new TachyonURI(Utils.getPathWithoutScheme(path));
+    Path hdfsPath = Utils.getHDFSPath(tPath, mUnderFSAddress);
+    LOG.info("setPermission({},{}) HDFS Path: {} TPath : {}{}", path, permission.toString(),
```

Comment:
About LOG.debug
I think we should keep the same log level with other commands.
TPath -> Tachyon Path
Thanks, I will also updated `getFileStatus`